### PR TITLE
Add GetAxialSlices() method

### PR DESCRIPTION
**Pull request by woswos**
Add GetAxial() method for axial geometry type.

### CODE REVIEW

File: include/openrave/kinbody.h [hunk 1]
```diff
@@ -768,7 +768,17 @@ class OPENRAVE_API KinBody : public InterfaceBase
         inline int GetNumberOfAxialSlices() const {
             return _info._vAxialSlices.size();
         }
-
+        inline std::vector<dReal> GetAxial() const {
```
[the remark this answers is not in the document]
Comment (rdiankov) [reply include/openrave/kinbody.h:771]: I'm not sure I agree with this approach. I think bindings should just return a list of structs with `zOffset` and `radius` as members.

[the remark this answers is not in the document]
Comment (woswos) [reply include/openrave/kinbody.h:771]: I will call it `GetAxialSlices` and return a list structs

[the remark this answers is not in the document]
Comment (woswos) [reply include/openrave/kinbody.h]: cc: @ziyan

[the remark this answers is not in the document]
Comment (ziyan) [reply include/openrave/kinbody.h]: @rdiankov This is quite a weird design:

1. We have a flatten list of numbers in JSON
2. Our API/Frontend wants to edit the whole flatten list altogether
3. We have a different representation in memory and in python binding

Why don't we just follow the example of `SideWall` then? Right now, this new structure is in between a `SideWall` and `TriMesh`, creating a brand new convention in the code. And we are all confused on when it is a flatten list and when it is a list of struct?

**Pull request closed without merging**